Apps/Examples/Examples/All Examples/Annotations/PointAnnotationClusteringExample.swift

```diff
@@ -93,20 +93,15 @@ class PointAnnotationClusteringExample: UIViewController, ExampleProtocol {
             UIColor.lightPink
         }
 
-        // Create a cluster property to add to each cluster feature
-        // This will be added to the cluster textField below
-        let clusterProperty: [String: Expression] = ["pointString": Exp(.string) { "Count:\n" }]
-
         // Select the options for clustering and pass them to the PointAnnotationManager to display
         let clusterOptions = ClusterOptions(circleRadius: .expression(circleRadiusExpression),
                                             circleColor: .expression(circleColorExpression),
                                             textColor: .constant(StyleColor(.black)),
                                             textField: .expression(Exp(.concat) {
-                                                Exp(.get) {"pointString"}
+                                                Exp(.string) { "Count:\n" }
                                                 Exp(.get) {"point_count"}
                                             }),
-                                            clusterRadius: 75,
-                                            clusterProperties: clusterProperty)
```

Comment:
the clustering behavior is removed, which will not reflect the example's name: PointAnnotationClustering.  Maybe still keep the `clusterProperties` by adding a sum property, even though it is functioning the same as `point_count`

Reply:
Without `clusterProperties` there is still clustering behavior (the default is `nil`). I do like the idea of showing all of the options though so I'll add one back in.